src/broker/Manager.h

```diff
@@ -346,15 +346,9 @@ class Manager : public iosource::IOSource {
 		__attribute__((format (printf, 2, 3)));
 
 	// IOSource interface overrides:
-	void GetFds(iosource::FD_Set* read, iosource::FD_Set* write,
-	            iosource::FD_Set* except) override;
-
-	double NextTimestamp(double* local_network_time) override;
-
 	void Process() override;
-
-	const char* Tag() override
-		{ return "Broker::Manager"; }
+	const char* Tag() override	{ return "Broker::Manager"; }
+	double GetNextTimeout() override	{ return -1; }
```

Comment:
Maybe a similar situation/concern to `DNS_Mgr`: if we have no activity on the FDs, is there something that will drain Broker's `pending_queries` or does it grow indefinitely until there is activity?  Maybe need to set the timeout here so we get a chance to kick out expired data store queries ?  (Maybe the old implementation even lacked this)

Reply:
I don't see anywhere where we directly flush the pending queries in the broker code other than some calls to `TrackStoreQuery` in broker/store.bif. I have no idea how often those get called.

That said, adding a timer that fires periodically to force them to flushed isn't a ton of work. What's a reasonable timeout value?

Reply:
Sorry and nevermind the original concern: think it's fine the way it is since Broker/CAF are going to guarantee a response comes in eventually no matter what (even if the remote store/process dies, it sends an error response) and that will end up triggering a process via the data store's FD and then erasing stuff from `pending_queries`.  Kind of weird how it's decoupled from the script-layer timeout, but the processing of the response takes note of that and things should work out fine.